src/config/package-documentation/index.md

```diff
@@ -0,0 +1,16 @@
+# Package documentation
+
+The most common format of documentation in Void Linux are [manual
+pages](./man.md).
+
+Many packages contain documentation in other formats, like HTML. This
+documentation can usually be found in a `/usr/share/doc/<package>` directory.
+
+More extensive documentation may be split into separate `*-doc` packages, such
+as `julia-doc`. This is often the case for programming languages, databases and
+big software libraries.
+
+In addition to documentation provided by upstream authors, packages may also
+contain some provided by Void project. This typically describes initial setup,
+or usage specific to Void. Location of it is a
+`/usr/share/doc/<package>/README.voidlinux`.
```

Comment:
```suggestion
In addition to documentation provided by upstream authors, packages may also
contain additional documentation provided by Void Linux (or "the Void project"). This typically describes initial setup,
or usage specific to Void. It can be located in
`/usr/share/doc/<package>/README.voidlinux`.
```

Reply:
"In addition ... additional" sounds poor.

"Void project" is already a term in docs, do we want to be that verbose here?

What about "It will" rather that "It can"?

Reply:
> What about "It will" rather that "It can"?

Yes

> "Void project" is already a term in docs, do we want to be that verbose here?

It's ok to use it, but then it needs "the" before it (what I suggested in parentheses).

> "In addition ... additional" sounds poor.

Hmm, true. Maybe "In addition to documentation provided by upstream projects, packages may also
contain other documentation provided" ? (I think switching from "authors" to "upstream" should happen either way).

Reply:
Actually, I don't like "other" either.

Reply:
reworded